kingpin/actors/rightscale/api.py

```diff
@@ -462,9 +473,34 @@ def wait_for_task(self, task, task_name=None, sleep=5, logger=None):
 
             yield utils.tornado_sleep(min(sleep, 5))
 
+        log.debug('Task (%s) status: %s (updated at: %s)' %
```

Comment:
The logger is only for the actor, and other logs are api directly. And being an optional value I can't reliably use it.

If you want to generalize this such that an actor instance is required when instantiating the API object then I can write it